NotToDo/NotToDo/Presentation/AddMissionScene/Views/AddMissionTextField.swift

```diff
@@ -32,25 +28,33 @@ final class AddMissionTextFieldView: UIView {
 
 // MARK: - Methods
 
-extension AddMissionTextFieldView {
+extension AddMissionTextField {
     private func setUI(placeHolder: String) {
-        AddMissionTextField.do {
+        self.do {
             $0.backgroundColor = .nottodoWhite
             $0.layer.borderWidth = 1.adjusted
             $0.layer.borderColor = UIColor.nottodoGray4?.cgColor
             $0.font = .PretendardMedium(size: 16)
             $0.leftView = UIView(frame: CGRect(x: 0.0, y: 0.0, width: 15.adjusted, height: 0.0))
             $0.leftViewMode = .always
             $0.attributedPlaceholder = NSAttributedString(string: placeHolder, attributes: [NSAttributedString.Key.foregroundColor: UIColor.nottodoGray3!])
+            $0.addTarget(self, action: #selector(changeText), for: .editingChanged)
```

Comment:
```suggestion
backgroundColor = .nottodoWhite
layer.borderWidth = 1.adjusted
layer.borderColor = UIColor.nottodoGray4?.cgColor
font = .PretendardMedium(size: 16.adjusted)
leftView = UIView(frame: CGRect(x: 0.0, y: 0.0, width: 15.adjusted, height: 0.0))
leftViewMode = .always
attributedPlaceholder = NSAttributedString(string: placeHolder, attributes: [NSAttributedString.Key.foregroundColor: UIColor.nottodoGray3!])
addTarget(self, action: #selector(changeText), for: .editingChanged)
```
여기서는 이렇게 바꿔주셔도 좋을 것 같습니다!! 클래스 자체가 UITextField이기 do를 사용하지 않고도 바로바로 속성 적용해주면 좋을 듯 해유!!

Reply:
오오 넵!! 근데 이렇게 했더니 addTarget부분에서 오류가 뜨는데... 이유를 알 수 있을까요ㅠㅠ

Reply:
어라 저도 다시 한 번 해볼게유